PowerEditor/src/Notepad_plus.cpp

```diff
@@ -6095,6 +6129,10 @@ void Notepad_plus::launchProjectPanel(int cmdID, ProjectPanel ** pProjPanel, int
 			(*pProjPanel)->openWorkSpace(nppParam.getWorkSpaceFilePath(panelID));
 	}
 	(*pProjPanel)->display();
+
+	checkMenuItem(cmdID, true);
+	checkProjectMenuItem();
+	(*pProjPanel)->setClosed(false);
 }

```

Comment:
This last three new lines in **launchProjectPanel** are actually the same as in **launchFileBrowser**: Check the menu item and set **_isClosed** to **false**.